contents/security/sqlInjection.md

```diff
@@ -60,30 +94,28 @@ AND Password = ‘pw1234’
 This query would find all entries in the `Users` table in the database, and return any entries where the `Username` is `Alice` and the `Password` is `pw1234`. If the previous table was searched, it would return the details of the second row as the result. Since there was a result returned, the website would be able to tell that a legitimate username and password combination was entered since the query requires that both are matched to retrieve the data. Thus, the website would know that the user is legitimate, and the user would be allowed to log into the site. On the other hand, if no result was returned by the database, the website would know that the username and password combination does not match any of the users in the database, and would deny access to the person trying to log in.
 
 
-However, are users limited to just filling in legitimate username and password strings? Some websites directly use the information given by the user without any validation, thus the users are able to write code as input for their own malicious purposes. For example, users are able to add parameters to the query.
+However, users are not limited to just fill in legitimate username and password strings. Some websites directly use the information given by the user without any validation. Thus an attacker is able to write code as input for their own malicious purposes. For example, the attacker can add  more parameters to the query:
 
 ```sql
-Username: foo
-Password: bar’ OR ‘’=’
+Username: Admin
+Password: foo’ OR ‘1’=’1
```

Comment:
I've not used SQL so I'm not sure about this, but logically shouldn't it be `foo’ OR ‘1’=‘1`? (The last apostrophe is an opening quotation mark)